redash/models/__init__.py

```diff
@@ -1132,7 +1135,6 @@ def all(cls, org, group_ids, user_id):
                 ),
                 Dashboard.org == org,
             )
-            .distinct()
```

Comment:
Intentional @rafawendel?

Lead to this issue: https://github.com/getredash/redash/issues/5466

Reply:
Hello @susodapop! Any updates? Did you submit your changes?

Reply:
Yes

Reply:
Thanks! I'm now able to reproduce. Working on a patch now.

Reply:
For posterity, the reason this call was removed is because SQLA raises a comparison error when making a generic `distinct()` because dashboards includes a JSON field...

@Tri0L I've opened a PR that fixes this in case you want to test it out: https://github.com/getredash/redash/pull/5516